packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -756,7 +756,9 @@ flutter:
         FakeCommand(
           command: <String>[
             impellerc,
-            '--sksl',
+            // --sksl is implicit.
```

Comment:
For these - should I make it explicit in the tool? I'm not sure what will be easier to deal with going forward.